protocol-units/bridge/contracts/src/AtomicBridgeCounterparty.sol

```diff
@@ -39,19 +43,21 @@ contract AtomicBridgeCounterparty is IAtomicBridgeCounterparty, OwnableUpgradeab
         bytes32 originator,
         bytes32 bridgeTransferId,
         bytes32 hashLock,
-        uint256 timeLock,
         address recipient,
         uint256 amount
     ) external onlyOwner returns (bool) {
         if (amount == 0) revert ZeroAmount();
         if (atomicBridgeInitiator.poolBalance() < amount) revert InsufficientWethBalance();
 
+        // The timelock for the counterparty is just 24 hours (constant value)
+        uint256 timeLock = block.timestamp + TIME_LOCK_DURATION;
+
         bridgeTransfers[bridgeTransferId] = BridgeTransferDetails({
             recipient: recipient,
             originator: originator,
             amount: amount,
             hashLock: hashLock,
-            timeLock: block.number + timeLock, // using block number for timelock
+            timeLock: timeLock,
```

Comment:
Since counterparty duration is fixed as 24 hours, would it make sense to name the constant COUNTERPARTY_TIME_LOCK_DURATION on the counterparty side? And INITIATOR_TIME_LOCK_DURATION on the initiator side, so its' crystal clear which time lock duration is which?

Reply:
That's been added